plugins/microsoft_iis.yaml

```diff
@@ -28,15 +28,18 @@ pipeline:
     include:
       - {{ $file_path }}
     multiline:
-      line_start_pattern: '\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2} '
+      line_start_pattern: '\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2} |#Software: |#Version: |#Date: |#Fields: '
     start_at: {{ $start_at }}
     labels:
       log_type: microsoft_iis
       plugin_id: {{ .id }}
-    output: microsoft_iis_parser
+
+  - type: filter
+    expr: '$record matches "#Software: |#Version: |#Date: |#Fields: "'
```

Comment:
Why are we filtering and doing an if condition? Are we intentionally dropping all the startup logs?

Reply:
The if statement helps in cases of failed parsing I believe. It is possible that if other fields are selected for logging we will not parse correctly and this would add the log entry to the record without parsing. 

I am filtering the startup logs due to causing parse errors, but with the if on the regex it would add them to record and not cause parsing errors if we removed the filter. This could be removed without issue if we wanted to capture those logs.